src/uncommenter.rs

```diff
@@ -76,6 +81,8 @@ fn basic_tests() {
     assert_eq!(uncomment("//123\n45"), "     \n45");
     assert_eq!(uncomment("//123\n//45\n6"), "     \n    \n6");
 
+    assert_eq!(uncomment("#ifdef foo\nblah\n#endif\ngrah"), "          \nblah\n      \ngrah");
```

Comment:
Please also add tests for a `#` inside a `//` comment and a `#` inside a `/**/` comment. Those are two cases that do appear in the code base. I'm pretty sure it'll work based on reading the code but it would be nice to have that covered by a test.